### ISSUE TITLE
Add alt text

### ISSUE THREAD

**Issue by sierraobryan** [WomenWhoCodeCincy/flutter-starter-app#20]
Use `semanticLabel: 'Alt text'` 

Examples of what should have alt text: 
- image: what is the image of? (example: Image of member's name)
- buttons: what does the button do? (example: navigate to name of link) 
- icons: what is the icon of? (example: twitter icon)

Some examples of what does not need alt text: 
- text widgets 
- parent container
